src/lib/alarmMgr/AlarmManager.h

```diff
@@ -83,7 +83,7 @@ class AlarmManager
   bool forwardingError(const std::string& url, const std::string& details);
   bool forwardingErrorReset(const std::string& url);
 
-  bool badInput(const std::string& ip, const std::string& details);
+  bool badInput(const std::string& ip, const std::string& details, const std::string& extraInLog);
```

Comment:
I could have opted for using `const std::string& extraInLog = ""` here and save a lot of changes along the code.

However, I have preferred to raise all the places in which badInput() is used, to analyze in which places it makes sense to add extra log info.

Once the work has been done, maybe now  `const std::string& extraInLog = ""`  can be used here and all the places in which we have  `alarmMgr.badInput(..., "")` changed to just `alarmMgr.badInput(...)` so reducing "noise".

Reply:
Fixed in [b33022b](https://github.com/telefonicaid/fiware-orion/pull/4130/commits/b33022b45c64fbe3ad035c2dab4ca272d1eb090d)